src/site/content/en/blog/web-otp-iframe/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Fill an OTP form within a cross-origin iframe using WebOTP API
+subhead: WebOTP API can now receive an OTP from within an iframe.
+authors:
+  - yigu
+  - agektmr
+date: 2021-04-21
+hero: image/YLflGBAPWecgtKJLqCJHSzHqe2J2/cjCJPMZpWEK9MgnH3MK1.jpg
+alt: A man entering the credit card number on a computer.
+
+description: |
+  WebOTP API can now receive an OTP from within an iframe.
+
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - identity
+  - capabilities
+  - otp
+  - iframe
+  - cross-origin
```

Comment:
```suggestion
  - blog # blog is a required tag for the article to show up in the blog.
  - identity
  - capabilities
```

Reply:
Removed unsupported tags. you can find a list of valid tags here: https://github.com/GoogleChrome/web.dev/blob/main/src/site/_data/tagsData.json

You can always submit a PR to add new tags along with reasons why they should be added.